### DISCUSSION TITLE
How change MySQL config `ft_min_word_len` in PlanetScale?

### DISCUSSION THREAD

**Discussion by lazyfishkk** [planetscale/discussion#548] [Q&A]
> [ft_min_word_len](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_ft_min_word_len): The minimum length of the word to be included in a MyISAM FULLTEXT index.

For searches in some non-English languages (such as Chinese, Japanese...), which are often searched by two characters (Chinese characters), such as "你好".

```SQL
FROM posts WHERE MATCH(title) AGAINST('你好')
```
> Currently I always return 0 results with SQL like this, even if there is matching data.

Maybe I should modify `ft_min_word_len=2` _(MySQL Default=4)_  to better support full text search.

Can PlanetScale allow me to support 2-character full-text indexing?

**Comment by jgreet** [comment]
Hello,

PlanetScale databases use MySQL 8 and the InnoDB storage engine, so the variable you'd need for is probably [innodb_ft_min_token_size](https://dev.mysql.com/doc/refman/8.0/en/innodb-parameters.html#sysvar_innodb_ft_min_token_size) not `ft_min_word_len`. Unfortunately `innodb_ft_min_token_size` is not configurable for PlanetScale databases at this time.

See also discussion https://github.com/planetscale/discussion/discussions/364
